tests/Rules/LocalOnlyPublicClassMethodRule/Fixture/SkipPublicCallbackMethod.php

```diff
@@ -18,5 +18,6 @@ private function run()
 
 function doFoo()
 {
-    call_user_func([SkipPublicCallbackMethod::class, 'runHere']);
+    $object = new SkipPublicCallbackMethod();
```

Comment:
This test was wrong. It was using the callback format that should be used to call static functions. But `runHere` is not static. The new PHPStan detected this problem and I fixed it by making a non-static call

Reply:
should this be a separate PR?

Reply:
Maybe, not sure it is worth it